test/e2e/Dockerfile

```diff
@@ -20,52 +20,59 @@ RUN apt-get update && \
 RUN mkdir -p /go/src/k8s.io
 WORKDIR /go/src/k8s.io
 
+### Kubernetes 1.25
+FROM builder AS tests-1.25
+ARG KUBE_VERSION_1_25=1.25.2
+ARG KUBE_VERSION_1_25_E2E_BIN_SHA256_CHECKSUM=b9ec7494abed4d5d40f4027e4211da0a2b33f76f81f6813430a9ebd142ca83a0
+
+RUN curl --fail --location https://dl.k8s.io/v${KUBE_VERSION_1_25}/kubernetes-test-linux-amd64.tar.gz | tar xvzf - --strip-components 3 kubernetes/test/bin/e2e.test kubernetes/test/bin/ginkgo
+RUN echo "${KUBE_VERSION_1_25_E2E_BIN_SHA256_CHECKSUM}" e2e.test | sha256sum --check
+RUN cp e2e.test /e2e.1.25.test
+RUN cp ginkgo /ginkgo-1.25
```

Comment:
Given we now require Kubernetes version-specific ginkgo binaries, is there a need to extend our "how to support a new Kubernetes minor version" documentation in the repo?

Reply:
Updated the e2e/Readme.md to say copy version specific ginkgo into final runner image.